contracts/prize-strategy/multiple-winners/MultipleWinners.sol

```diff
@@ -61,9 +72,76 @@ contract MultipleWinners is PeriodicPrizeStrategy {
     return __numberOfWinners;
   }
 
+  function setPrizeSplit(MultipleWinnersPrizeSplit[2] memory prizeStrategySplit) external onlyOwner {
+    for (uint256 index = 0; index < prizeStrategySplit.length; index++) {
+        MultipleWinnersPrizeSplit memory split = prizeStrategySplit[index];
+
+        // If MultipleWinnersPrizeSplit is non-zero address set the prize split configuation.
+        if (split.target != address(0)) {
+            // Split percentage must be below 1000 and greater then 0 (e.x. 200 is equal to 20% percent)
+            // The range from 0 to 1000 is used for single decimal precision (e.x. 15 is 1.5%)
+            require(
+                split.percentage > 0 && split.percentage < 1000,
```

Comment:
So we can't have `0` or `100` percent of the prize splitted?
I guess it wouldn't be a split anymore but maybe people would like to redirect 100% of the prize to a single address.

Reply:
Added the ability split 100% of the prize. But keeping the inability to have a prize split with 0% as it would simply be a waste of gas to iterate on that MultipleWinnersPrizeSplit config.